packages/formstr-app/src/containers/CreateFormNew/components/AnswerSettings/index.tsx

```diff
@@ -61,12 +74,49 @@ function AnswerSettings() {
     editQuestion(field, field[1]);
   };
 
+  const handleImageUpload = (markdownUrl: string) => {
+    const name = markdownUrl.match(/\[(.*?)\]/)?.[1] || '';
+    const url = markdownUrl.match(/\((.*?)\)/)?.[1] || '';
+    const imageMarkdown = `![${name}](${url})`;
+  
+    const newImages = [...uploadedImages, { name, url }];
+    setUploadedImages(newImages);
+    localStorage.setItem(`uploadedImages_${questionIdInFocus}`, JSON.stringify(newImages));
```

Comment:
why do we need to set this in localStorage?

Reply:
yes, we don't need local-storage, as it is already getting saved. i was also thinking about persisting the data in reloads, but it is starting with a clean slate rn, so we need to set it up on a bigger scale if we need that.